src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -158,10 +163,10 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 
 	void handleMultiPadPress(ModelStackWithTimelineCounter* modelStack, InstrumentClip* clip, int32_t firstPadX,
 	                         int32_t firstPadY, int32_t secondPadX, int32_t secondPadY, bool modEncoderAction = false);
+	void renderDisplayForMultiPadPress(ModelStackWithTimelineCounter* modelStack, InstrumentClip* clip,
+	                                   int32_t xDisplay = kNoSelection, bool modEncoderAction = false);
 
 	int32_t calculateKnobPosForModEncoderTurn(int32_t knobPos, int32_t offset);
-	bool isOnParameterGridMenuView();
```

Comment:
renamed isOnParameterGridMenuView() to is isOnAutomationOverview() and made the function public as its used by view.cpp